src/documents/APIChangesFrom1_5To2_0.md

```diff
@@ -0,0 +1,136 @@
+---
+title: API Changes from 1.5 to 2.0
+layout: default
+---
+
+# API Changes from 1.5 to 2.0 #
+
+## Framework Changes ##
+
+### Preferences Framework ###
+
+#### Namespace Changes ####
+
+Rename "fluid.prefs.enactors" to "fluid.prefs.enactor"
+
+#### Component Grade Changes ####
```

Comment:
The thing though is that this was an explicit change made to the prefsFramework that changes the way it is used now. There is documentation in for the model relay system in the infusion docs already. https://github.com/fluid-project/infusion-docs/blob/master/src/documents/ModelRelay.md